03-other-topics/006-prometheus.md

```diff
@@ -57,6 +57,82 @@ AlertManager: [https://alertmanager.apps.cloud-platform-live-0.k8s.integration.d
 
 Grafana: [https://grafana.apps.cloud-platform-live-0.k8s.integration.dsd.io](https://grafana.apps.cloud-platform-live-0.k8s.integration.dsd.io)
 
+## Custom Rules/Alerts
+
+Once you own a namespace, you can have custom alerts setup that can send alert information to your slack channel when triggered.
+
+Steps:
+1. Create a ticket with the Cloud Platform team requesting a new Alermanager Alert Route be set up and include the following information:
+  
+- Namespace
+- Team Name
+- Application Name
+- Slack Channel messages shoud go to
+
+Once completed, the Cloud Platform team will supply you with a `value` for the `severity` label. All alerts are routed using this label.
+
+2. Create a PrometheusRule
+
+#### prometheus-custom-rules-<application_name>.yaml
+
+Once the route configuration is complete by the Cloud Platform Team, use the 'severity' label value supplied and create a custom rule using the template below:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  namespace: <namespace>
+  labels:
+    prometheus: prometheus-operator
+    role: alert-rules
+  name: prometheus-custom-rules-<application_name>
```

Comment:
Do we want to enforce a naming scheme for this? It's going to live in a user namespace after all.

Reply:
I would like to, as during testing, its possible to add a PromRule which break the AM rules config from refreshing. Going through logs, it would make it quicker to troubleshoot than see something like ~"my-rules" config has undefined annotations~ etc